app/views/wiki/_quick_navigation.html.erb

```diff
@@ -0,0 +1,46 @@
+<%
+  actions = YAML.load(File.read(Rails.root.join("config/arrays/wiki", "actions.yml"))).map { |_, c| c }.sort_by { |c| c["en-US"] }
+  values = YAML.load(File.read(Rails.root.join("config/arrays/wiki", "values.yml"))).map { |_, c| c }.sort_by { |c| c["en-US"] }
+  events = YAML.load(File.read(Rails.root.join("config/arrays/wiki", "events.yml"))).map { |_, c| c }.sort_by { |c| c["en-US"] }
+  constants = YAML.load(File.read(Rails.root.join("config/arrays/wiki", "constants.yml"))).map { |n, c| n }.sort_by { |n| n }
+%>
+
+<div class="quick-navigation" data-role="dynamic-max-height">
+  <div class="quick-navigation__scroll" data-role="sticky" data-dynamic-height-target data-scroll-container>
+    <% { "Actions": actions, "Values": values, "Events": events }.each do |category, props| %>
+      <div data-toggle-content>
+        <button class="quick-navigation__toggle <%= "active" if @article.category.title == category.to_s %>" data-action="toggle-content">
+          <%= category %>
+
+          <div class="quick-navigation__icon">&#8250;</div>
+        </button>
+
+        <div data-role="content-to-toggle" style="<%= "display: none" if @article.category.title != category.to_s %>">
+          <% props.each do |content| %>
+            <%= link_to content["en-US"],
+              CGI.escape(content["en-US"]).downcase,
+              class: "quick-navigation__item #{ "quick-navigation__item--active" if @article.title == content["en-US"] }",
+              data: { prefetch: false, role: ( 'scroll-into-view-on-load' if @article.title == content["en-US"]) } %>
```

Comment:
It might be nice to make the active item not a link at all to avoid allowing users to reload the page by accidentally clicking on it.

Reply:
I see what you mean, but I'd rather keep it as is, even if only to make it not unnecessarily complex.

Reply:
Fair enough